ctapipe/tools/apply_models.py

```diff
@@ -201,31 +201,18 @@ def _apply(self, reconstructor):
 
             for tel_id, table in chunk.items():
                 tel = self.loader.subarray.tel[tel_id]
-
-                if isinstance(reconstructor, DispReconstructor):
-                    if tel not in reconstructor._norm_models:
-                        self.log.warning(
-                            "No norm regressor for telescope type %s, skipping tel %d",
-                            tel,
-                            tel_id,
-                        )
-                        continue
-                    if tel not in reconstructor._sign_models:
-                        self.log.warning(
-                            "No sign classifier for telescope type %s, skipping tel %d",
-                            tel,
-                            tel_id,
-                        )
-                        continue
-                else:
-                    if tel not in reconstructor._models:
-                        self.log.warning(
-                            "No model in %s for telescope type %s, skipping tel %d",
-                            reconstructor,
-                            tel,
-                            tel_id,
-                        )
-                        continue
+                if (
+                    tel not in reconstructor._norm_models
```

Comment:
If you'd use a tuple of (disp_model, sign_model) in a self._models dict, the API of the different reconstructors would basically be the same here

Reply:
Good idea, I changed it.